src/api_server.py

```diff
@@ -10,84 +9,23 @@
 from fastapi import FastAPI
 from loguru import logger
 from pydantic import BaseModel, validator
-from torch.utils.data import DataLoader
 
-from src.data import QuestionAnsweringInputEncoder, QuestionAnsweringModelInput
-from src.models.qa import QuestionAnsweringModel
-from src.stop_watch import time_block
+from src.model_extractors import BenefitsAndDrawbacksExtractor
 
 
 @dataclass
 class ApiServerConfig:
     prediction_log_path: str = "./data/logs/predictions/logs.jsonl"
     api_log_path: str = "./data/logs/api/logs.txt"
-    model_name: str = "choosistant/qa-model"
+    model_qa: str = "choosistant/qa-model"
+    model_seq2seq: str = "choosistant/seq2seqmodel"
```

Comment:
Consider be explicit with variable name to enhance readibility of the code. Suggestion change variables `model_qa` to `model_name_qa` and `model_seq2seq` to `model_name_seq2seq`,

Reply:
Done ! thank you for the tip